README.md

```diff
@@ -1 +1,55 @@
-# ruby-enumerable
\ No newline at end of file
+# ruby-enumerable
+
+It is a project about building methods with same functionality as the enumerable methods
+
+- #my_each
+- #my_each_with_index
+- #my_select
+- #my_all?
+- #my_any?
+- #my_none?
+- #my_count
+- #my_map
+- #my_inject
+
+# What it does
+
+Uses loops and other syntax to provide the same output as if using the original enumerable methods
+
+## Demo
+
+This is a link to demo
+Repl.it(https://repl.it/@okerioh/enumarable-methods )
```

Comment:
You can make this a hyperlink by writing this: `[Repl.it](https://repl.it/@okerioh/enumarable-methods)`